torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1339,6 +1341,71 @@ def call_function(
         )
 
 
+class StrictModeHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        from .builder import wrap_fx_proxy
+        callable = args[0]
+        graph_checkpoint, checkpoint = tx.output.graph, tx.copy_graphstate()
+        (ret_val, ret_treespec), ret_graph, ret_lifted_freevars, = speculate_subgraph(
+            tx,
+            args[0],
+            args[1].unpack_var_sequence(tx),
+            {},
+            "strict_mode",
+            source_target=self.value,
+            manually_set_subgraph_inputs=False,
+            should_flatten_outputs=True,
+        )
+
+
+        strict_mode_nn_modules = tx.copy_graphstate().output.nn_modules
+
+        strict_mode_name = add_subgraph(
+            tx,
+            self.source,
+            "strict_mode_body",
+            torch.fx.GraphModule(strict_mode_nn_modules.nn_modules, ret_graph),
+        )
+
+
+        strict_mode_node = make_attr(tx, strict_mode_name)
```

Comment:
I remember some time ago, we want to inline the graph into top-level instead of creating a call_function nodes. Do we change the plan?

Reply:
yes we will in the next PR!